packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -383,12 +386,14 @@ export async function activate(context: vscode.ExtensionContext) {
     sfdxProjectOpened
   );
 
+  // register org picker commands and set up filewatcher for defaultusername
+  const orgList = new OrgList();
+  context.subscriptions.push(registerOrgPickerCommands(context, orgList));
+  await orgList.registerDefaultUsernameWatcher(context);
```

Comment:
I think we should only register the file watcher if the cli is installed. What do you think ?

Reply:
I think that makes sense- I believe we should also only register the org picker commands if the cli is present as well. None of the methods triggered by the filewatcher directly use the cli anymore, but later on the org picker does use `force:config:set` which will fail without the cli